sentences/da/_common.yaml

```
@@ -46,10 +106,176 @@ lists:
       - "fahrenheit"
       - in: "f"
         out: "fahrenheit"
+
+  # binary_sensor
+  bs_battery_states:
+    values:
+      - in: "lav"
+        out: "on"
+      - in: "normal"
+        out: "off"
+  bs_battery_charging_states:
+    values:
+      - in: "lader"
+        out: "on"
+      - in: "lader ikke"
+        out: "off"
+  bs_carbon_monoxide_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_cold_states:
+    values:
+      - in: "koldt"
+        out: "on"
+      - in: "normalt"
+        out: "off"
+  bs_connectivity_states:
+    values:
+      - in: "forbundet"
+        out: "on"
+      - in: "afbrudt"
+        out: "off"
+  bs_door_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_garage_door_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_gas_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_heat_states:
+    values:
+      - in: "varmt"
+        out: "on"
+      - in: "normalt"
+        out: "off"
+  bs_light_states:
+    values:
+      - in: "detekteret"
+        out: "on"
+      - in: "ikke detekteret"
+        out: "off"
+  bs_lock_states:
+    values:
+      - in: "ulåst"
+        out: "on"
+      - in: "låst"
+        out: "off"
+  bs_moisture_states:
+    values:
+      - in: "vådt"
+        out: "on"
+      - in: "tørt"
+        out: "off"
+  bs_motion_states:
+    values:
+      - in: "(detekteret|påvist)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist)"
+        out: "off"
+  bs_occupancy_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
+        out: "on"
+      - in: "(ikke detekteret|ikke påvist|fraværende)"
+        out: "off"
+  bs_opening_states:
+    values:
+      - in: "åben"
+        out: "on"
+      - in: "lukket"
+        out: "off"
+  bs_plug_states:
+    values:
+      - in: "tilkoblet"
+        out: "on"
+      - in: "frakoblet"
+        out: "off"
+  bs_power_states:
+    values:
+      - in: "tændt"
+        out: "on"
+      - in: "slukket"
+        out: "off"
+  bs_presence_states:
+    values:
+      - in: "(hjemme|tilstedeværende)"
+        out: "on"
+      - in: "(væk|fraværende)"
+        out: "off"
+  bs_problem_states:
+    values:
+      - in: "fundet"
+        out: "on"
+      - in: "intet fundet"
+        out: "off"
+  bs_running_states:
+    values:
+      - in: "(kører|aktiveret)"
```

I don't think "aktiveret" makes sense for this.

```suggestion
      - in: "kører"
```